### DISCUSSION TITLE
Make average windspeed average

### DISCUSSION THREAD

**Discussion by Nikuksis** [beyond-all-reason/rfcs#9] [Gameplay and Mechanics]
The average wind speed in BAR is not really an arithmetic average of min and max wind speed of a map. Which is confusing the newbies and actually impact energy balance a lot. Also wrong formulas used in other places in the game e.g. buildmenu disables windgens on maps with 0-10 wind, while actually it is still better then solars there.

I want for wind more transparent mechanics, which are obvious for everyone. Also allow map designs with really risky winds like (1-25) and more stable winds (5-10).
Current mechanics are tesnd to stick to the maximum at the most part of gametime and confuse people with it's average speeds.
Also this will make windfarms riskier so more downtime will appear for them. That will force players make more energy storages and gives more diversity in the overall T1 eco. People have to decide for themself how much windgens should be in their T1 ec and also punish people with greedy 100% wind ecoing by the increasing probability of stalling.

**Comment by Teifion** [comment]
I think this is a very sensible idea. I would suggest the implementation details be updated to include an action to update all maps to have new wind-speed values such that the actual balance is maintained.

**Comment by Nikuksis** [comment]
I'm up for increasing windspeed in compensation on the maps if needed.

**Reply by Teifion** [reply]
My thoughts are it'd counteract any balance complaints which I imagine would be the most likely complaints to come out of this.

**Comment by WatchTheFort** [comment]
I think we're approaching wind design the wrong way, but that discussion is outside the scope of this RFC. In short, it doesn't matter how we model wind, what matters is the game design intent.

Wind is supposed to be a cheaper, but less reliable source of power. In that view, wind should always be viable on every map, the player should not have to do _any_ analysis or memorization. There are some maps where we want wind to be disabled for flavour (some space maps), so for that it can be zero.

**Reply by Teifion** [reply]
If it's outside the scope of the RFC, please don't comment on it.

**Comment by erpiederp** [comment]
Teifion wrote: "The RFC represents a specific and defined proposal. It will be reviewed by the game design team (which is currently being decided on from amongst the established contributors) but they'll take into consideration comments on it."

This RFC isnt specific or defined. @Nikuksis could you add to it so that it becomes clear what exactly you propose?

**Reply by Nikuksis** [reply]
Added a bit of details into RFC md
